_includes/comparison.html

```diff
@@ -200,7 +200,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>Kubernetes</td>
         <td>Kubernetes</td>
         <td>ECS, Fargate, EKS, EC2</td>
-        <td>Kubernetes, Nomad, VMs (Universal)</td>
+        <td>ECS, Kubernetes, Nomad, VMs (Universal)</td>
```

Comment:
This seems to be a tech preview, see https://www.consul.io/docs/ecs . Should there be a note concerning its state?

Reply:
I added a callout that this is in tech preview.